index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="container">
+      <!-- navgator -->
+
+      <nav>
+        <div class="logo">
+          <a href="#">Welcome</a>
+        </div>
+
+        <!-- menu hamburger -->
+        <a href="#"><img class="menu" src="imege/Enabled.png" alt="Menu" /></a>
+      </nav>
```

Comment:
- [ ] Please add your logo inside a `<a>` tag and put your hamburger button inside a `<nav>` tag and kindly put both of them inside a `<header>` tag because it will be better semantically that way.